sample-templates/anomaly-detector-suggestion-agent-claude.json

```diff
@@ -0,0 +1,113 @@
+          "type": "create_tool",
+          "previous_node_inputs": {
+            "register_claude_model": "model_id"
+          },
+          "user_inputs": {
+            "parameters": {
+              "model_type":"",
+              "prompt": "Human:\" turn\": Here are some examples of the create anomaly detector API in OpenSearch: Example 1. POST _plugins/_anomaly_detection/detectors, {\"time_field\":\"timestamp\",\"indices\":[\"ecommerce\"],\"feature_attributes\":[{\"feature_name\":\"feature1\",\"aggregation_query\":{\"avg_total_revenue\":{\"avg\":{\"field\":\"total_revenue_usd\"}}}},{\"feature_name\":\"feature2\",\"aggregation_query\":{\"max_total_revenue\":{\"max\":{\"field\":\"total_revenue_usd\"}}}}]}, Example 2. POST _plugins/_anomaly_detection/detectors, {\"time_field\":\"@timestamp\",\"indices\":[\"access_log*\"],\"feature_attributes\":[{\"feature_name\":\"feature1\",\"feature_enabled\":true,\"aggregation_query\":{\"latencyAvg\":{\"sum\":{\"field\":\"responseLatency\"}}}}]} and here are the mapping info containing all the fields in the index ${indexInfo.indexName}: ${indexInfo.indexMapping}, and the optional aggregation methods are value_count, avg, min, max and sum, note that value_count can perform on both numeric and keyword type fields, and other aggregation methods can only perform on numeric type fields. Please give me some suggestion about creating an anomaly detector for the index ${indexInfo.indexName}, you need to give the key information: the top 3 suitable aggregation fields which are numeric types(long, integer, double, float, short etc.) and the suitable aggregation method for each field, you should give at most 3 aggregation fields and corresponding aggregation methods, if there are no numeric type fields, both the aggregation field and method are empty string, and also give at most 1 category field if there exists a keyword type field whose name is just like region, country, city or currency, if not exist, the category field is empty string, note the category field must be keyword type. Show me a format of keyed and pipe-delimited list wrapped in a curly bracket just like {category_field=the category field if exists|aggregation_field=comma-delimited list of all the aggregation field names|aggregation_method=comma-delimited list of all the aggregation methods}. \n\nAssistant:\" turn\""
+            },
+            "name": "CreateAnomalyDetectorTool",
+            "type": "CreateAnomalyDetectorTool"
+          }
+        },
+        {
+          "id": "anomaly_detector_suggestion_agent",
+          "type": "register_agent",
+          "previous_node_inputs": {
+            "create_anomoly_detectors_tool": "tools"
+          },
+          "user_inputs": {
+            "parameters": {},
+            "type": "flow",
+            "name": "Anomaly detector suggestion agent",
+            "description": "this is the anomaly detector suggestion agent"
+          }
+        }
+      ],
+      "edges": [
```

Comment:
We don't need `edges` if previous node inputs are provided correctly. Can you remove edges and test the template again and confirm?

Reply:
Thanks, I've removed edges and tested the new template yet, it works fine, please take a look.